mlkem/ntt.c

```diff
@@ -90,10 +91,11 @@ __contract__(
   ensures(array_abs_bound(r, 0, MLKEM_N - 1, (layer + 1) * MLKEM_Q - 1)))
 {
   /* `layer` is a ghost variable only needed in the CBMC specification */
+  int start;
```

Comment:
Misplaced declaration, should not be between comment and `((void) layer`

Reply:
resolved